lib/component-repository/src/routes/virtual-components/GetTrigger.js

```diff
@@ -1,3 +1,34 @@
+const mongoose = require('mongoose');
+const ComponentVersion = require('../../models/ComponentVersion');
+
 module.exports = async function (req, res, next) {
-    return next();
+  const { componentVersionId, triggerName } = req.params;
+
+  if (!mongoose.Types.ObjectId.isValid(componentVersionId)) {
+    return res
+      .status(400)
+      .send({ errors: [{ message: 'Invalid id', code: 400 }] });
+  }
+
+  const componentVersion = await ComponentVersion.findOne(
+    {
+      _id: componentVersionId,
+      'triggers.name': triggerName,
+    },
+    {
+      'triggers.$': 1,
+    }
+  ).lean();
+
+  if (!componentVersion.triggers[0]) {
```

Comment:
The same that I commented with checking the action

Reply:
Done